share/spice/frequency_allocation/frequency_allocation.js

```diff
@@ -0,0 +1,62 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_frequency_allocation = function(api_result) {
+
+        if (!api_result || api_result.status == 'Fail' || api_result.Errors) {
+            return Spice.failed('frequency_allocation');
+        }
+        
+        var script = $('[src*="/js/spice/frequency_allocation/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/frequency_allocation\/([^\/]+)/)[1],
+            targetFrequency = decodeURIComponent(query);
+        
+        
+        var infoboxData;
+        if (targetFrequency >= 3000 && targetFrequency < 30000){
+            infoboxData = [{heading: 'ITU assignment'},
+                                  {label: "Designation", value: 'Super high frequency (SHF)'},
+                                  {label: "Description", value: 'The SHF band is used by microwave ovens, wireless LANs, cell phones, and satellites.'},
+                                  {label: "Lower band", value: 3000 + ' MHz'},
+                                  {label: "Upper band", value: 30000 + ' MHz'}]     
+        } else if(targetFrequency >= 300 && targetFrequency < 3000){
+            infoboxData = [{heading: 'ITU assignment'},
+                                  {label: "Designation", value: 'Ultra high frequency (UHF)'},
+                                  {label: "Description", value: 'The UHF band is used by televisions, cordless phones, cell phones, pagers, walkie-talkies, and satellites.'},
+                                  {label: "Lower band", value: 300 + ' MHz'},
+                                  {label: "Upper band", value: 3000 + ' MHz'}]
+        } else if (targetFrequency >= 30 && targetFrequency < 300){
```

Comment:
@gaulrobe. You right. But in the situation i think that keeping the real contant is better because I don't know if i will need to extend the range of the trigger in the backend. Then if i have to do, i only have to modify the backend code but not the frontend. It's the same with the maximum contant '30000'.